corehq/apps/hqwebapp/migrations/0005_auto_20170426_2139.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-04-26 21:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqwebapp', '0004_auto_20170423_1750'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='usedpasswords',
+            old_name='password',
+            new_name='password_hash',
+        ),
+    ]
```

Comment:
Would it be safe now to merge them since the 2 models are being now removed? Those tables are already present at least on staging.